src/core/api/clock.ts

```diff
@@ -336,12 +336,12 @@ function createClock(
 
     function getCurrentClockTick(state : IMediaInfosState) : IClockTick {
       const mediaTimings = getMediaInfos(mediaElement, state);
+      log.debug("API: current media element state", mediaTimings);
```

Comment:
Here, it's the clock minus `stalled` and `getCurrentTime`. Wouldn't we want at least the former for a tick log?